sdk/cosmosdb/cosmos/package.json

```diff
@@ -0,0 +1,75 @@
+{
+  "name": "@azure/cosmos",
+  "description": "Azure Cosmos DB Service Node.js SDK for SQL API",
+  "keywords": [
+    "cosmosdb",
+    "cosmos db",
+    "documentdb",
+    "document database",
+    "azure",
+    "nosql",
+    "database",
+    "cloud"
+  ],
+  "version": "2.1.5",
+  "author": "Microsoft Corporation",
+  "main": "./lib/src/index.js",
+  "types": "./lib/src/index.d.ts",
+  "engine": {
+    "node": ">=6.0.0"
+  },
+  "scripts": {
+    "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
+    "clean": "rimraf lib",
+    "lint": "tslint --project tsconfig.json",
+    "format": "prettier --write --config .prettierrc.json \"src/**/*.ts\"",
+    "check-format": "prettier --list-different --config .prettierrc.json \"src/**/*.ts\"",
+    "compile": "echo Using TypeScript && tsc --version && tsc --pretty",
+    "compile-prod": "echo Using TypeScript && tsc --version && tsc -p tsconfig.prod.json --pretty",
+    "prepack": "npm install && npm run build",
+    "pack": "npm pack 2>&1",
+    "webpack": "webpack -d",
+    "webpack-prod": "webpack -p",
+    "build": "npm run clean && npm run check-format && npm run lint && npm run compile && npm run webpack",
+    "build-prod": "npm run clean && npm run check-format && npm run lint && npm run compile-prod && npm run webpack-prod",
+    "test": "mocha -r ./src/test/common/setup.ts ./lib/src/test/ --recursive --timeout 100000 -i -g .*ignore.js",
+    "test-ts": "mocha -r ts-node/register -r ./src/test/common/setup.ts ./src/test/**/*.spec.ts --recursive --timeout 100000 -i -g .*ignore.js",
+    "ci": "npm run build && npm run test && node ts-test.js"
+  },
+  "devDependencies": {
+    "@types/mocha": "^5.2.5",
+    "@types/node": "^8.10.22",
+    "@types/priorityqueuejs": "^1.0.1",
+    "@types/semaphore": "^1.1.0",
+    "@types/sinon": "^4.3.3",
+    "@types/tunnel": "^0.0.0",
+    "@types/underscore": "^1.8.8",
+    "execa": "1.0.0",
+    "mocha": "^5.2.0",
+    "mocha-junit-reporter": "^1.18.0",
+    "mocha-multi-reporters": "^1.1.7",
+    "prettier": "^1.16.4",
+    "requirejs": "^2.3.5",
```

Comment:
Looks like we're missing `"rimraf": "^2.6.2",` as a devDependency - it's needed because some of the scripts use the rimraf CLI tool.

Reply:
Added.